spec/meilisearch/index/settings_spec.rb

```diff
@@ -722,4 +722,65 @@ def update_synonyms(index, synonyms)
       expect(index.pagination.transform_keys(&:to_sym)).to eq(default_pagination)
     end
   end
+
+  context 'On typo tolerance' do
+    let(:index) { client.index(uid) }
+
+    let(:default_typo_tolerance) do
+      {
+        'enabled' => true,
+        'minWordSizeForTypos' =>
+         {
+           'oneTypo' => 5,
+           'twoTypos' => 9
+         },
+        'disableOnWords' => [],
+        'disableOnAttributes' => []
+      }
+    end
+
+    let(:new_typo_tolerance) do
+      {
+        'enabled' => true,
+        'minWordSizeForTypos' => {
+          'oneTypo' => 6,
+          'twoTypos' => 10
+        },
+        'disableOnWords' => [],
+        'disableOnAttributes' => ['title']
+      }
+    end
+
+    before { client.create_index!(uid) }
+
+    it 'get default typo tolerance settings' do
+      task = index.typo_tolerance
+      client.wait_for_task(task['taskUid'])
+
+      expect(index.typo_tolerance).to eq(default_typo_tolerance)
+    end
```

Comment:
The `/typo-tolerance` sub-route does not respond with a task :)

```suggestion
    it 'gets default typo tolerance settings' do
      settings = index.typo_tolerance

      expect(settings).to eq(default_typo_tolerance)
    end
```